seeds.sql

```diff
@@ -0,0 +1,29 @@
T81bDlQ1jIDWEcy7YDxBBtZseO6oTZIfJ7Nrqc2uLxsXQKv6PM5uaxlgR+7xhxoGzuRe23XzQLHx3GW6PETZvuW17OaQJZjMfgBa2ujy2yB6c7/lZj+10fP8ATH/G5TmKNjA0ULrxJPM+Kkau95RxUZUpUZQDgnJjSnBAccuBdK4EA4JJJIBpXF0pID4rCL6bKhAVnFkopcp0bGtZDJsoM42FVgn2TpJLUVjYgtFpXIIBGEV06WktrY1LGe6qGW1XcQ2FHlR7IUihGEpjsuckyR2yWqwIzwg0vNGc0oLKd02JM1vGctT2Z1n2UjSfynfxbyP0tZPGVmJxa4FERr1fLk4M2Hi2iPvFw6tokb910tXnZUM0fR4HMEX15A9Csv2VlblQxcW74j7N18yyvdPy29Ctf/8Am4wjLGhrX/Itt18QHOt1mM7sNcpqWsFqfZ7Gc4llxHrTuIcudFDsfShA9rmOFg2CNvvZaLW+z0YL3MleyqIJdYOxu+gG33SAa5jnG4y2YPYxvG72pDSN6B25A7pvC0eeP1WlzMs8IPeEIE9qr2YyJsiKTjYRG3eJ3SQcrb1o/tslXC6kskh5lRONytNaqeKFdakyUhs8nA0uq6BIHeegQXHy3ykmmlraDrd73GRfw9B4+BR2Ye6UPhwSCC0c3Di3IAbRs1e/T73WSz7ZbfpaxMV2TK0M+GhvyAHW1v8AT8ZsbQ1vwtFeZ/MUD0aPhrhHThFC6JHM/IfNaSIUKXTw4zW3Nzclt19JCutXEmqznSqNykUb0BxqkCjanhAdKak54C5xIBy6mgrpcgEVxJ7qTWutAfFqkjKiUjEARhkU/GqURUxKXwUmQhA5XMeSiqOALRB0O1qGc1VJWl07JFKxkyilmcPKI2V/8QSFkh8ae+UKCWRQuJTHFLYtKp5rkKI3RPJVHg3Wyp51bxY1OWbqPFcrTt1kvabV9i9Q9jMw37jiGP7uEnn6HdeoZuQ1jvetjh8DwAQAed+HeF4nprt17y/EZNGC7mWg3/4qmU9WMxy0xGt/hHm5RFIducYs14Wepv0WZl03F4+NsLS67Bc1po9KHoPktdq+hMbZDRfQ8ItCXaeGbuNnoPBJ5fytLNdRYw5QyN2+5AQkyW71UuS/ZVYeaX+RBjHdsrTZKQsZQaOaqZmuRRbyyNZ3AkcR9OZWa23bQPl2UT8tkbS+R4jYNyXENHr/AAsHqXb9o2gjLz/mktrB4hvM+tLI6hqs2S7ilkL65A7MZ/paNgqY8NvvpLLmk9dvauxfa/8AF5roIwPw4iLmlwp75WvbuP7aJ2PcvSWL5w/wz1IQZ8BJ917vYn/2AtB8rIX0exdWMmM1HNbvtIuArqrzOpDFsOTXFUG5CRyd0BdapAoIXWp0AI1zJdGAQL33Tm5BLL8Faz4Q5pQ6Hax4IasNyTw+Kk/EbNvqhuTJRaO9LU5+AwjvdSwG9uNSdBjucz4+Tf8AUeSf2a1Ay47Hu+IjfzrdR9pmB5ja7kT9VZ07T+BlDldptzQ+nyKnNTU4IYtxlT81TjcrkO6eME9LajIZsh+nR0ilLh5r26cMegyqeUSxmKnMze1fwjyWY1sSvxVDLjo02Owop4hSKrKzWTGqEjEZzGobI1ITJUZLSuY81qlLGmseQu7g+NeSbQyz00OHLRC9Z0LtO10LGuPvNaGmzzAFA/JeGMzaWl0PVgaFpuT41xZjnt6Lq2vsBJ4t+gtZnK1ppJJcPClbxseB+7oo3HvMbCf0Wc/xDxgPZPYODZzbYA0dCBt6rmnFNq3l1Cy+0cI5vBPhug+X2xYNmAu8v5WWlxwdySSqb4yFT8UTvNaNZvaeeT4T7Mf27u/3Hl6UhBcSbJJJ5kkkn1KjCcCnkk9Ett9pLT2lRNUrFrE7TQPlS+puyRf+CxfaO45Pw8PG6+K3+ybdnqV8sMctL2U7VZODKJI5HeysccLnExPbe44Ts011FFAfTaikZai0vPZkwxzxkmORjZGWKPC4XuOhVhYEQgTH4yuNScgIYm0pkxPCAgyBsULiPvFO17UhGAOpNDzKbjMIaCeqGqWqg+1i81S7UOPtceuQfZ8uEozqENvYe5VdSg433/l3RKEfbCJ7scSR/Gynt8a3r15eqJ6HniWFj+8A/RTxsD4KPKkF0P3Yy0cg9wHla3fQfKi6FxdCGJGIlht3Q6JGcGPknk62J7GMJqvuCrQMoK0F5vLd5O3f6quWKUUGTwlSZrtkEmmort+Jwfkc2efi2cGcKTJ8q1mIMwonjS2url+J4TZZy7OyBaqFiIy1SG5MtLhnH5Zah7l0glCoPVh8yqOcvf8Ai8Xhj25c8to3lPwskscDajcoit5cdjG6ejaNqlgbovqEbcmF0bt7BLf7X17pC8y07OLCtdpmqXW68nl4vGr43bGuiINHY+QVeZnedvQBGtVjDXPPTiNeRNoLK0uO/oOiRNSLb5DZdERRCKH1Ur2UEabsOaxPpSOCZSxpzFZaVXapmID0fsB28fhBkM1vxDY73QkuJJHe3e6XtmPO2RoexwcxwDmuabBB5EL5ZjdbSO7deif4T9rTFIMOV39KQ/0S4/8ATkP5fI/r5rA9qYulNYnFARlPCY5PagA+fpIlka478JseavTwUAArgCTggKGRFy8AqjY74z4Is5trgiG/isALJMRiuI5gFU+yWM78Kwv+J3vO8zutC7EBaW9CpIYQwBoGwW760HxcuhJJaE8CO4J5JJLrwwl4y77GGSbJ5moJJLybjPN0y9B2XkWhEj7K4kvZ+FjI5uWpYn0iONk0upLt5cJlO0pdJps1DMma0kkvDwYStuVQFyauJLqI4VG5JJR5DxwFF9LnISSXFzTpXD2lypC41Q9d1C3FvnskkvO2bScMDRsqWW8HZJJDFRwTSF1JDSapmJJLWJ4zScHlpBBog2CNiCORCSSA+j+wWu/jsSOQn+oP6cv/AHG9fUUfVaNJJYEbk5qSSAcF0pJLAaUkkkAkkkkB/9k=');
```

Comment:
I had to look these up to find out about data URIs...cool way to embed a small profile pic!